src/sftp/handler.go

```diff
@@ -26,13 +26,13 @@ type SftpHandler struct {
 func NewSftpHandler(credentialGetter secrets.CredentialGetter) (*SftpHandler, error) {
 	// In the future, we'll pass in info about what customer we're using (and thus what URL/key/password to use)
 
-	// TODO uncomment code when partner is setup to receive key
-	//pem, err := getPublicKeysForSshClient(credentialGetter)
-	//if err != nil {
-	//	return nil, err
-	//}
+	pem, err := getPublicKeysForSshClient(credentialGetter)
+	if err != nil {
+		slog.Error("Unable to get public keys for ssh client", slog.Any("KeyName", pem), slog.Any(utils.ErrorKey, err))
+		return nil, err
+	}
 
-	serverKeySecret := "sftp-server-public-key-" + utils.EnvironmentName() // pragma: allowlist secret
+	serverKeySecret := "ca-phl-sftp-server-public-key-" + utils.EnvironmentName() // pragma: allowlist secret
```

Comment:
It might be worth refactoring `ca-phl` out into a constant somewhere that we plug into these strings. 

This is another spot where I might put `Name` somewhere in the variable name to indicate that this value is just the name of the secret and not the secret itself

Reply:
Refactored and renamed!